### PR TITLE
Bumped NuGet packages in NuGetGallery.FunctionalTests

### PR DESCRIPTION
**Pull request by aseemsahoo**
The versions of the NuGet packages in `NuGetGallery.FunctionalTests` solution have been updated.

### CODE REVIEW

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9198) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9198) before we can accept your contribution.<br/><br/>:x: aseemsahoo [sign now](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9198)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/NuGet/NuGetGallery?pullRequest=9198) it.</sub>

**Comment by joelverhagen** [comment]
Thanks for the contribution @aseemsahoo! I'll run this through our CI and DEV environment to make sure everything is working as expected.

**Comment by joelverhagen** [comment]
I've rebased this branch on latest `dev` since we merge all non-hotfixes into `dev` prior to `main`.

File: tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj [hunk 1]
```diff
@@ -72,25 +72,25 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder">
-      <Version>2.2.0</Version>
+      <Version>6.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Web.Xdt">
-      <Version>2.1.1</Version>
```
Comment (joelverhagen) [inline tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj:78]: This appears to be causing this error when running functional tests:
```
Warning: Test Run deployment issue: The assembly or module 'Microsoft.Web.XmlTransform' directly or indirectly referenced by the test container 'd:\a\_work\1\s\tests\nugetgallery.webuitests.p2\bin\release\nugetgallery.webuitests.p2.dll' was not found.
```

Comment (joelverhagen) [reply tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj:78]: Would it be possible to remove this direct dependency and just let `NuGet.Core` pull in the default version transitively?

**Review by joelverhagen — CHANGES_REQUESTED** [review]
The Microsoft.Web.Xdt update is causing problems.

**Pull request closed without merging**